doc/user/content/transform-data/patterns/temporal-filters.md

```diff
@@ -9,25 +9,48 @@ menu:
     parent: 'sql-patterns'
 ---
 
-A _temporal filter_ is a `WHERE` or `HAVING` clause which uses the [`mz_now()`](/sql/functions/now_and_mz_now) function.
-This function returns Materialize's current virtual timestamp, which works to keep up with real time as data is processed.
-Applying a temporal filter reduces the working dataset, saving memory resources and focusing results on the recent past.
-
-Here is a typical temporal filter that considers records whose timestamps are within the last 5 minutes.
+A **temporal filter** is a query condition/predicate that filters data based on
+a time-related (i.e., temporal) condition. Using a temporal filter reduces the
+working dataset, saving memory resources and focusing on results that are within
+the timeframe specified by the filter.
+
+In Materialize, you implement temporal filters using the
+[`mz_now()`](/sql/functions/now_and_mz_now) function (which returns
+Materialize's current virtual timestamp and keeps up with real time) in a
+`WHERE` or `HAVING` clause; specifically, you compare
+[`mz_now()`](/sql/functions/now_and_mz_now) to a numeric or timestamp column
+expression. As [`mz_now()`](/sql/functions/now_and_mz_now) progresses (every
+millisecond), records that fall outside the timeframe are retracted from the
```

Comment:
I don't think there is a "timeframe" to fall outside of. The way I think of this is that as `mz_now()` advances, the predicate involving (usually) data columns and `mz_now()` may start to or cease to be true. When these changes happen, Materialize translates them into data insertion and removal.

Reply:
Will update.